ngDraggable.js

```diff
@@ -574,23 +574,39 @@ angular.module("ngDraggable", [])
 
                         if (config.horizontalScroll) {
                             // If horizontal scrolling is active.
-                            if (lastMouseEvent.clientX < config.activationDistance) {
+
+                            // lastMouseEvent.clientX is undefined when dealing with a touch device, resulting in
+                            // no scrolling when dragging an item to the bottom of the screen
+                            // Seen on Chrome 47.0.2526.111
+                            var clientX = lastMouseEvent.clientX;
+                            if (angular.isUndefined(lastMouseEvent.clientX))
+                              clientX = lastMouseEvent.touches[0].clientX;
```

Comment:
touches is not defined on event itself, it is defined on originalEvent - use lastMouseEvent.originalEvenet.touches[0]
or rather - inject ngDraggable and use ngDraggable.inputEvent(lastMouseEvent).clientY